test/check-avocado-selenium

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# This file is part of Cockpit.
+#
+# Copyright (C) 2015 Red Hat, Inc.
+#
+# Cockpit is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# Cockpit is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+
+import os
+import sys
+import unittest
+
+import testvm
+from testlib import *
+
+avocado_tests = ["selenium-login.py"]
+
+
+@unittest.skipIf(True, "Selenium tests aren't yet reliable")
```

Comment:
Lets merge the infrastructure then. If we're not actually running the tests though, there's no point in having this file. Someone literally has to check this file out, modify it, in order to get it to run. That's identical to having to checkout another branch or pull request.